src/api-reference/request/examples.markdown

```diff
@@ -0,0 +1,893 @@
+---
+title: Examples
+layout: reference
+---
+
+# Request examples
+
+Below we have sample request data and the corresponding request answers:
+
+* [expense-resource](#expense-resource)
+* [request-resource](#request-resource)
+* [travel-agency-resource](#travel-agency-resource)
+* [workflow-resource](#workflow-resource)
+
+
+<a name="expense-resource"></a>
+### expense-resource
```

Comment:
Expense-resource or Expense resource in order to be consistent with the Get Started page